packages/react/src/components/Button/Button.js

```diff
@@ -119,17 +121,21 @@ const Button = React.forwardRef(function Button(
   }, []);
 
   const enabled = useFeatureFlag('enable-2021-release');
+  console.log('feature flag is:', enabled);
 
   const buttonClasses = classNames(className, {
     [`${prefix}--btn`]: true,
     [`${prefix}--btn--sm`]: size === 'small' || size === 'sm' || small,
     [`${prefix}--btn--md`]: size === 'field' || size === 'md',
     // V11: change lg to xl
-    [`${prefix}--btn--lg`]: enabled ? size === 'xl' : size === 'lg',
+    [`${prefix}--btn--lg`]: enabled
+      ? size === 'xl'
+      : size === 'lg' || isExpressive,
     // V11: change xl to 2xl
     [`${prefix}--btn--xl`]: enabled ? size === '2xl' : size === 'xl',
```

Comment:
@jnm2377 @shixiedesign are xl (soon to be 2xl) buttons allowed to have an expressive variant?

Reply:
The 2xl buttons are for tearsheets. I would say yes for for now.